src/components/awesome_link.cr

```diff
@@ -22,4 +29,25 @@ class AwesomeLink < BaseComponent
       end
     end
   end
+
+  private def render_tags
+    (tags || [] of Tags).each do |tag|
+      if tag.adult?
+        span "18+",
```

Comment:
How about `Adult` instead of `18+`? Age leans more into the legal aspect (in particular) of pornography (in particular), in the united states (in particular). Even then the age varies state to state. To address a world-wide audience and themes other than pornography, I think `Adult` could be both more correct and help inform the reader so they can make the best decision for themselves.

What concept do you want to communicate with this text?

Reply:
That's true... and actually, that might make more sense anyway. I did a "beer and wine" online store a few years back, and that would be one that would fall in to this category (as well as sites like dispensaries or whatever...). Alcohol related sites generally need a 21+ anyway (at least in the U.S.). Good call.